tests/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:20.10
+
+ADD ./runner /
+ADD ./tracee-trainer/* /
+RUN chmod +x /runner
+
+RUN apt-get update
+
+# TRC-3 Code injection
```

Comment:
could we have one script (and layer) that sets up all the dependencies?

Reply:
Updated https://github.com/aquasecurity/tracee/pull/1282/commits/6d3be669a2e7ce253508aa49560260168816f4bf